dev/com.ibm.ws.install/test/com/ibm/ws/install/ServerAssetTest.java

```diff
@@ -96,6 +100,29 @@ public void testGetServerAssetName() throws Exception {
         }
     }
 
+    @Test
+    public void testGetServerFeaturesAndPlatforms() throws Exception {
+        initializeTestCase("testGetServerFeaturesAndPlatforms");
+
+        File sServerXML = new File("publish/servers/serverA/server.xml");
+        File dServerXML = new File(testDir, "server.xml");
+
+        try {
+            copyFile(sServerXML, dServerXML);
+            assertTrue("Cannot find server.xml file: " + dServerXML.getCanonicalPath(), dServerXML.isFile());
+
+            ServerAsset sa = new ServerAsset(dServerXML);
+            File serverXmlFile = sa.getServerXmlFile();
+            FeaturesPlatforms fp = InstallUtils.getFeatures(serverXmlFile.getAbsolutePath(), serverXmlFile.getName(), new HashSet<String>());
+            assertTrue(fp.getFeatures().containsAll(Arrays.asList("genericCoreFeature", "featureA-1.0", "featureB-1.0", "featureC-1.0")));
+            assertTrue(fp.getPlatforms().containsAll(Arrays.asList("jakartaee-10.0")));
```

Comment:
`publish/servers/serverA/server.xml` should include 
```
<platform>jakartaee-10.0</platfrom>
```

Reply:
I think I did this here: https://github.com/OpenLiberty/open-liberty/pull/28981/files#diff-ca27070527c748105825085f9de6650f3dce9bdbf3434559531e6e531bbb61b0

Reply:
and this test does run in the build so we'll know soon if it passes. :)

Reply:
sorry missed that earlier